webrender/src/device.rs

```diff
@@ -341,6 +341,8 @@ impl TextureId {
             target: gl::TEXTURE_2D,
         }
     }
+
+    pub fn is_invalid(&self) -> bool { *self == TextureId::invalid() }
```

Comment:
while certainly useful, I think it would be better to avoid double-negative, and have `is_valid()` method instead